pkg/devspace/config/configutil/get.go

```diff
@@ -500,3 +509,55 @@ func GetDefaultNamespace(config *latest.Config) (string, error) {
 
 	return "default", nil
 }
+
+// GetContextAjustedConfig returns the config with setting the correct values for namespace and kubeContext based on generated config and on args provided
+func GetContextAjustedConfig(namespace, kubeContext string) (*latest.Config, error) {
+	// load generated config
+	generatedConfig, err := generated.LoadConfig()
+	if err != nil {
+		return nil, errors.Errorf("Error loading generated.yaml: %v", err)
+	}
+
+	// load config
+	config, err := GetConfigFromPath(".", generatedConfig.ActiveConfig, true, generatedConfig, log.GetInstance())
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
The generated config has to be saved in this function otherwise entered variables will not be saved.

Reply:
Makes sense. Was not obvious from the code. Maybe this should generally done after setting all variables?

Reply:
Done for now as requested.